app/controllers/settings_controller.rb

```diff
@@ -45,8 +45,12 @@ def edit
       end
 
       settings.each do |name, value|
-        # remove blank values in array settings
-        value.delete_if(&:blank?) if value.is_a?(Array)
+        if value.is_a?(Array)
+          # remove blank values in array settings
+          value.delete_if(&:blank?)
+        else
+          value = value.strip
```

Comment:
I cannot assume a reason why a trailing whitespace is required in any case. can you?